napalm_ios/ios.py

```diff
@@ -78,6 +79,23 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
         self.config_replace = False
         self.interface_map = {}
 
+    def file_copy(self, src, dest=None, file_system='flash:'):
+        """
+        SCP file to device filesystem, filesystem defaults to flash:.
+
+        Return None or raise exception
+        """
+        if dest is None:
+            dest = basename(src) 
+        if src:
+            (return_status, msg) = self.scp_file(source_file=src,
```

Comment:
If we are going to standardize on file_copy in NAPALM, we can probably just rename scp_file to be file_copy. We can leave a reference to scp_file (pointing to file_copy) so if anyone uses the old name, it doesn't break anything.

I would think you would want both a file put and file get operation (i.e. get file from remote device and put file to remote device).